### PR TITLE
weechat: add page

### PR DESCRIPTION
**Pull request by 4G3NT**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
3.8

### CODE REVIEW

File: pages/common/weechat.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# weechat
+
+> Command-line internet relay chat client with various features.
+> More information: <https://weechat.org>.
```
Comment (acuteenvy) [inline pages/common/weechat.md:4]:
```suggestion
> More information: <https://weechat.org/doc>.
```

File: pages/common/weechat.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
```
Comment (acuteenvy) [inline pages/common/weechat.md:5]:
```suggestion

- Start `weechat`:

`weechat`

```

**Review by acuteenvy — APPROVED** [review]

**Comment by 4G3NT** [comment]
I find the start `[command]` thing weird some pages don't even use it take a look at krita's tldr page

**Comment by 4G3NT** [comment]
I could make a pull request to change to the \`command\` convention, but I don't think it is a good change since the descriptions won't render properly.

**Comment by 4G3NT** [comment]
You can list the pages that don't use the back-tick convention by doing:
```sh
tldr --list | while read -r m; do
	if tldr "$m" | grep -qiP "^- start $m"; then
		echo "$m"
	fi
done 2> /dev/null
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.
